Dockerfile

```diff
@@ -5,4 +5,4 @@ RUN mkdir -p /openqaoa
 
 ADD ./ /openqaoa/
 
-RUN pip3 install /openqaoa/.
+RUN cd openqaoa && make dev-install-tests
```

Comment:
do we need to install the `test` version of openqaoa from the Dockerfile? It doesn't seem like we were installing the `test` version before

Reply:
Good point. I'll try the normal install and see if tests work.